src/test/scala/com/wavesplatform/state/diffs/smart/predef/package.scala

```diff
@@ -49,7 +50,7 @@ package object predef {
        | let basic = longAll && sumString && sumByteVector && eqUnion
        | 
        | # 2) ne
-       | let nePrim = 1000 != 999 && "ha" +"ha" != "ha-ha" && tx.bodyBytes != base64'hahaha'
+       | let nePrim = 1000 != 999 && "ha" +"ha" != "ha-ha"
```

Comment:
wait, order should have bodybytes too, otherwise we won't be able to sigVerify.
maybe we should even add it prior to mainnet release.

Reply:
Dummy body bytes array added, but I need body bytes in orderV1/V2 to complete this.